### PR TITLE
Improve array processing in AbstractBlocks

### PR DESCRIPTION
**Pull request by szepeviktor**
Fix last PHPStan error.
Please see #124

### CODE REVIEW

**Review by dingo-d — DISMISSED** [review]
LGTM 👍🏼  Will have to test this on a project to be 100% sure it works as intended. Thanks!

File: src/Blocks/AbstractBlocks.php [hunk 1]
```diff
@@ -430,20 +430,20 @@ protected function getAttributes(array $blockDetails): array
...
-		} else {
-			$output = $componentAttributes;
+		if ($realComponentName === $newComponentName) {
+			return $componentAttributes;
 		}
 
-		return $output;
+		return array_reduce(
+			array_keys($componentAttributes),
+			function ($output, $name) use ($realComponentName, $newComponentName, $componentAttributes) {
```
Comment (szepeviktor) [inline src/Blocks/AbstractBlocks.php:441]: `static function` is a bit better, faster - I think `"php": "^7.2"` allows it.

Comment (dingo-d) [reply src/Blocks/AbstractBlocks.php:441]: Yeah, I think that PhpStorm suggests using `static` inside closures, so you can add it 👍🏼

Comment (szepeviktor) [reply src/Blocks/AbstractBlocks.php]: Done.

**Review by dingo-d — APPROVED** [review]

**Review by iruzevic — APPROVED** [review]

**Pull request merged**